### PR TITLE
[update] Enabling ACL

### PR DESCRIPTION
**Pull request by Rajakavitha1**
Addresses the issue: 
https://github.com/dgraph-io/dgraph-docs/issues/305
_added instructions to generate random key values

Note: Create and edit docs in the `main` branch when you can, so that we only cherry-pick out of `main`, not into `main`.
-->
Fixes: https://github.com/dgraph-io/dgraph-docs/issues/305

### CODE REVIEW

File: content/releases/index.md [hunk 1]
```diff
@@ -6,13 +6,10 @@ title = "Dgraph Releases"
...
 
-The latest Dgraph release is the v21.03 series.
+The latest Dgraph release is the v22.00 series.
 
-Dgraph releases starting with v20.03 follow
-[calendar versioning](https://calver.org). To learn more about our switch from
-semantic to calendar versioning, and why v2-v19 don't exist as a result of this
-switch, see our Blog post on the 
-[switch to calendar versioning](https://dgraph.io/blog/post/dgraph-calendar-versioning/).
+Dgraph releases starting v22.0.0 is following semver
```
Comment (MichelDiz) [inline content/releases/index.md:11]: Feels like this is a cherry pick? Why this changes are here? This is already in Main. And each PR should treat a single topic. Releases info are different from "Enabling ACL".

Comment (Rajakavitha1) [reply content/releases/index.md]: thanks for spotting this @MichelDiz !!! I have now fixed this.

**Review by skrdgraph — APPROVED** [review]
This looks good 👍  (apologies on delaying review here)

**Comment by Rajakavitha1** [comment]
> This looks good 👍 (apologies on delaying review here)

Thank you!!!!
